test/image/mocks/tickson_boundaries.json

```diff
@@ -38,12 +39,31 @@
       "orientation": "h",
       "xaxis": "x2",
       "yaxis": "y2"
+    },
+
+    {
+      "type": "bar",
+      "name": "with dtick !== 1",
+      "x": ["a", "b", "c", "d", "e", "f", "g", "h"],
+      "y": [1, 2, 1, 2, 1, 3, 4, 1],
+      "xaxis": "x3",
+      "yaxis": "y3"
+    },
+
+    {
+      "mode": "markers",
+      "marker": {"symbol": "square"},
+      "name": "with overlapping tick labels",
+      "x": ["A very long title", "short", "Another very long title"],
+      "y": [1, 4, 2],
+      "xaxis": "x4",
+      "yaxis": "y4"
```

Comment:
Nice, just squeaks in unrotated if we draw this without `tickson: 'boundaries'` 🎉 
![screen shot 2018-11-26 at 1 20 37 pm](https://user-images.githubusercontent.com/2678795/49033820-301a3f80-f17e-11e8-83a1-8378d7a36ab3.png)